app/controllers/loans_controller.rb

```diff
@@ -12,31 +12,48 @@ def new
       @user = User.find_by(id: params[:user_id])
       unless @user.good_to_borrow?
         flash[:error] = "User Can Not Borrow at This Time"
-        redirect_to user_path(@user.id) and return 
-      end 
+        redirect_to user_path(@user.id) and return
+      end
       @loan = Loan.new(user: @user)
     elsif params[:book_id]
       @book = Book.find_by(id: params[:book_id])
-      unless @book.available 
+      unless @book.available
         flash[:error] = "Book is Not Available"
-        redirect_to book_path(@book.id) and return 
-      end 
+        redirect_to book_path(@book.id) and return
+      end
       @loan = Loan.new(book: @book)
-    end 
+    end
+  end
+
+  def loan_multi
```

Comment:
This only loans books from the list until the patron hits their limit of 5, but doesn't have a way to convey that to user.